CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.32.0
+
+- Patch handling is more consistent. (#1263)
+- Command `packit generate` was removed. (#1269)
```

Comment:
We can maybe mention that it has been deprecated for a while in favour of `packit init`.